docs/source/Htypes.rst

```diff
@@ -533,36 +533,49 @@ Link htype
 
 >>> ds = deeplake.dataset("......")
 
-Add the names of the creds you want to use (not needed for http/local urls)
+:bluebold:`Adding credentials to the dataset`
+
+You can add the names of the credentials you want to use (not needed for http/local urls)
 
 >>> ds.add_creds_key("MY_S3_KEY")
 >>> ds.add_creds_key("GCS_KEY")
 
-Populate the names added with creds dictionary
-These creds are only present temporarily and will have to be repopulated on every reload
+and populate the added names with credentials dictionaries
 
 >>> ds.populate_creds("MY_S3_KEY", {})   # add creds here
 >>> ds.populate_creds("GCS_KEY", {})    # add creds here
 
-Create a tensor that can contain links
+These creds are only present temporarily and will have to be repopulated on every reload.
+
+For datasets connected to Activeloop Platform,
+`you can store your credentials on the platform <https://docs.activeloop.ai/storage-and-credentials/managed-credentials#managed-credentials-ui>`_ as Managed Credentials and 
+use them just by adding the keys to your dataset. For example if you have managed credentials with names ``"my_s3_creds"``, ``"my_gcs_creds"``, you can add them to your dataset using
+:meth:`Dataset.add_creds_key <deeplake.core.dataset.Dataset.add_creds_key>` without having to populate them.
+
+>>> ds.add_creds_key("my_s3_creds", managed=True)
+>>> ds.add_creds_key("my_gcs_creds", managed=True)
+
+
+:bluebold:`Create a link tensor`
 
 >>> ds.create_tensor("img", htype="link[image]", verify=True, create_shape_tensor=False, create_sample_info_tensor=False)
```

Comment:
@FayazRahman The flags should all be True. If Verify is True, there isn't much reason to set the others as False. Also, I'm not sure we should be including the flags at all in this point. Up to you.